java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/ArrowFlightJdbcAccessorTest.java

```diff
@@ -188,43 +188,43 @@ public void testShouldFailToGetBoolean() {
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetByte() {
+  public void testShouldFailToGetByte() throws SQLException {
     when(accessor.getByte()).thenCallRealMethod();
     accessor.getByte();
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetShort() {
+  public void testShouldFailToGetShort() throws SQLException {
     when(accessor.getShort()).thenCallRealMethod();
     accessor.getShort();
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetInt() {
+  public void testShouldFailToGetInt() throws SQLException {
     when(accessor.getInt()).thenCallRealMethod();
     accessor.getInt();
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetLong() {
+  public void testShouldFailToGetLong() throws SQLException {
     when(accessor.getLong()).thenCallRealMethod();
     accessor.getLong();
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetFloat() {
+  public void testShouldFailToGetFloat() throws SQLException {
     when(accessor.getFloat()).thenCallRealMethod();
     accessor.getFloat();
   }
 
   @Test(expected = UnsupportedOperationException.class)
-  public void testShouldFailToGetDouble() {
+  public void testShouldFailToGetDouble() throws SQLException {
     when(accessor.getDouble()).thenCallRealMethod();
     accessor.getDouble();
   }
 
   @Test(expected = UnsupportedOperationException.class)
```

Comment:
Is it correct? Are we expecting UnsupportedOperationExcetion,class?

Reply:
Done.